lib/slack/web/pagination/cursor.rb

```diff
@@ -7,23 +7,38 @@ class Cursor
 
           attr_reader :client
           attr_reader :verb
+          attr_reader :sleep_interval
+          attr_reader :max_retries
           attr_reader :params
 
           def initialize(client, verb, params = {})
             @client = client
             @verb = verb
+            @sleep_interval = params.delete(:sleep_interval)
```

Comment:
This modifies incoming `params`, ie. has a side-effect, you don't want to do that, so just `params[:sleep_interval]`.